### ISSUE TITLE
describe JCK-1

### ISSUE THREAD

**Issue by jjgomezcadenas** [next-exp/IC#80]
We are effectively moving towards JCK-1 syntax, and the last PR tries hard to follow the JCK notions. However, if we are going to ask other developers to follow it, we need a description/discussion (with examples, that can be taken from the Irene.py code) to illustrate the most important ideas.

**Comment by jacg** [comment]
> We are effectively moving towards JCK-1 syntax

It's not syntax. It's a style, or a guideline, but it's not syntax.
